content/hi/docs/reference/glossary/container-env-variables.md

```diff
@@ -0,0 +1,16 @@
+---
+title: कंटेनर वातावरण वेरिएबल्स (Container Environment Variables)
+id: container-env-variables
+date: 2018-04-12
+full_link: /docs/concepts/containers/container-environment/
+short_description: >
+  कंटेनर वातावरण वेरिएबल्स नाम = मान जोड़े हैं जो पॉड में चल रहे कंटेनरों में उपयोगी जानकारी प्रदान करते हैं।
```

Comment:
`कंटेनर  एनवायरनमेंट वेरिएबल्स नाम` will be more meaningful than `कंटेनर वातावरण वेरिएबल्स नाम here`